src/site/content/en/blog/top-cwv-2023/index.md

```diff
@@ -0,0 +1,203 @@
+
+The easiest way to solve this problem is to provide a hint to the browser about what resources are the highest priority by setting the new [`fetchpriority="high"`](/priority-hints/) attribute on the `<img>` or `<link>` tag that loads your LCP image. This instructs the browser to load it earlier, rather than waiting for those scripts to complete.
+
+According to the Web Almanac, only [0.03%](https://almanac.httparchive.org/en/2022/performance#lcp-prioritization) of eligible pages are taking advantage of this new API, meaning there is plenty of opportunity for most sites on the web to improve LCP with very little work. While the `fetchpriority` attribute is currently only supported in Chromium-based browsers, this API is a progressive enhancement that other browsers just ignore, so we strongly recommend developers use it now.
+
+For non-Chromium browsers, the only way to ensure the LCP resource is prioritized above other resources is to reference it earlier in the document. Using the example again of a site with lots of `<script>` tags in the `<head>` of the document, if you wanted to ensure your LCP resource was prioritized ahead of those script resources, you could add a `<link rel="preload">` tag before any of those scripts, or you could move those scripts to below the `<img>` later in the `<body>`. While this works, it's less ergonomic than using `fetchpriority`, so we hope other browsers add support soon.
+
+Another critical aspect of prioritizing the LCP resource is to ensure you don't do anything that causes it to be **deprioritized**, such as adding the `loading="lazy"` attribute. Today, [10%](https://almanac.httparchive.org/en/2022/performance#lcp-lazy-loading) of pages actually set `loading="lazy"` on their LCP image. Beware of image optimization solutions that indiscriminately apply lazy-loading behavior to all images. If they provide a way to override that behavior, be sure to use it for the LCP image. If you're not sure which image will be the LCP, try using heuristics to pick a reasonable candidate.
+
+Deferring non-critical resources is another way to effectively boost the relative priority of the LCP resource. For example, scripts that are not powering the user interface (like analytics scripts or social widgets) can be safely postponed until after the `load` event fires, which ensures they won't compete with other critical resources (such as the LCP resource) for network bandwidth.
+
+To summarize, you should follow these best practices to ensure that the LCP resource is loaded early, and at high priority:
+
+- **Add `fetchpriority="high"` to the `<img>` tag of your LCP image.** If the LCP resource is loaded via a` <link rel="preload">` tag, fear not because you can also set `fetchpriority="high"` on that!\
+- **Never set `loading="lazy"` on the `<img>` tag of your LCP image.** Doing this will deprioritize your image and delay when it starts loading.\
```

Comment:
```suggestion
- **Add `fetchpriority="high"` to the `<img>` tag of your LCP image.** If the LCP resource is loaded via a` <link rel="preload">` tag, fear not because you can also set `fetchpriority="high"` on that!
- **Never set `loading="lazy"` on the `<img>` tag of your LCP image.** Doing this will deprioritize your image and delay when it starts loading.
```

Reply:
Done